blockchain/fundamentals/ethereum-i/what-does-ethereum-do-differently.md

```diff
@@ -23,7 +27,37 @@ aspects:
 ---
 ## Content
 
+Other than being distributed and having a PoW algorithm initially, Ethereum does pretty much everything else *differently*.
+
+### Account
+
+Ethereum is using an **account model** aka balance model. You are probably familiar with the centralized version of this since it's employed by banks. The problem with a decentralized account model is its vulnerability to *replay attacks*: if you broadcast a signed transaction in which you pay Bob 0.5 ETH, Bob could re-broadcast the same transaction in order to receive 1 ETH in total. To prevent these attacks, Ethereum transactions have a *sequence number* (nonce).
```

Comment:
where would Bob re-broadcast? is it the exact same transaction ? do you want to say because that in a decentralised account model there's no central authority once can forge transactions which will fool the others into believing he has more money?

secondly, how does sequence numbers solve this specifically? myself, I'm curious about this

Reply:
Yeah, you could rebroadcast the same transaction multiple times being signed and everything. The nonce stops that by working as a counter: once a value is assigned to your account, any one of your transactions with a nonce less or equal that the one you hold is dropped.